topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.md

```diff
@@ -0,0 +1,321 @@
+
+Variation in the genome of *M. tuberculosis* (Mtb) is associated with changes in phenotype, for example [drug resistance](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-019-0660-8) and virulence. It is also useful for [outbreak investigation](https://www.frontiersin.org/articles/10.3389/fpubh.2019.00087/full) as the single nucleotide polymorphisms (SNPs) in a sample can be used to build a phylogeny.
+
+This tutorial will focus on identifying genomic variation in Mtb and using that do explore drug resistance and other aspects of the bacteria.
+
+# Get your data
+
+The data for today is a sample of *M. tuberculosis* [collected](https://www.ebi.ac.uk/ena/data/view/PRJEB6945) from a German [outbreak](https://journals.plos.org/plosmedicine/article?id=10.1371/journal.pmed.1001387) of tuberculosis. In addition to the bacterial sequence sample we will work with a Genbank format version of the genome of the inferred most recent common [ancestor](https://zenodo.org/record/3497110) of the M. tuberculosis complex which is combined with the annotation of the H37Rv reference sequence. This ancestral genome only differs from the H37Rv version 3 genome ([NC_000962.3](https://www.ncbi.nlm.nih.gov/nuccore/NC_000962.3)) by the insertion of SNPs to try and model the ancestor of all lineages of Mtb.
+
+This data is available at Zenodo using the following [link](http://doi.org/10.5281/zenodo.3531703).
+
+> ### {% icon hands_on %} Hands-on: Get the data
+>
+> 1. Import the following five files into a new history
+>   - [ERR550641_1.fastq.gz](https://zenodo.org/record/3531703/files/ERR550641_1.fastq.gz)
+>   - [ERR550641_2.fastq.gz](https://zenodo.org/record/3531703/files/ERR550641_2.fastq.gz)
+>   - [Mycobacterium_tuberculosis_ancestral_reference.gbk](https://zenodo.org/record/3531703/files/Mycobacterium_tuberculosis_ancestral_reference.gbk)
+>   - [Mycobacterium_tuberculosis_ancestral_reference.gff3](https://zenodo.org/record/3531703/files/Mycobacterium_tuberculosis_h37rv.ASM19595v2.45.chromosome.Chromosome.gff3)
+>   - [MTB_Ancestor_reference.fasta](https://zenodo.org/record/3497110/files/MTB_ancestor_reference.fasta)
+>
+>    {% include snippets/import_via_link.md %}
+>
+{: .hands_on}
+
+If you are using usegalaxy.eu for this tutorial, you can start by importing this [history](https://usegalaxy.eu/u/pvanheus/h/m-tuberculosis-variant-analysis-tutorial). Use the '+' button in the top right hand corner and select a meaningful name for the imported history before clicking 'Import'.
```

Comment:
I guess they will be in a data library too once merged

Reply:
Yep, I have removed this.